docs/news.rst

```diff
@@ -1076,7 +1076,7 @@ Cleanups & Refactoring
 ~~~~~~~~~~~~~~~~~~~~~~
 
 - Tests: remove temp files and folders (:issue:`2570`),
-  fixed ProjectUtilsTest on OS X (:issue:`2569`),
```

Comment:
I wonder if we should change occurrences in this file, or leave them as they are since the OS was named differently by the time those versions were released.

Reply:
It’s a valid point. 

I’m OK with the change, though. They all have the same [Wikipedia article](https://en.wikipedia.org/wiki/MacOS) :).

If we were mentioning a specific macOS version, then I think it would make sense to use the same branding they used for that specific version, though, unless they had rebranded it as macOS retroactively upstream.

Reply:
Yeah sounds fine to me, I was just thinking out loud.